geo_concerns/app/models/concerns/geo_concerns/image_behavior.rb

```diff
@@ -0,0 +1,31 @@
+module GeoConcerns
+  module ImageBehavior
+    extend ActiveSupport::Concern
+    include ::CurationConcerns::GenericWorkBehavior
+    include ::CurationConcerns::BasicMetadata
+    include ::GeoConcerns::BasicGeoMetadata
+
+    included do
+      aggregates :rasters, predicate: RDF::Vocab::ORE.aggregates,
+                           class_name: 'GeoConcerns::Vector',
+                           type_validator: type_validator
+      filters_association :members, as: :image_files, condition: :concerns_image_file?
```

Comment:
I just download a fresh copy of your repo, started a new gemset, and now the tests all pass. However, I'm not quite sure why. The image should aggregate rasters with class_name: 'GeoConcerns::Raster' rather than 'GeoConcerns::Vector' right? [spec/models/image_spec.rb](https://github.com/jrgriffiniii/pcdm-geo-models/blob/73f400f4ebd0be0ff3c2c92b5bb0c0f0047e8c1d/geo_concerns/spec/models/image_spec.rb#L30-L33) is where this is tested and where the failure happened for me.